CONTRIBUTING.md

```diff
@@ -196,32 +196,32 @@ https://numpydoc.readthedocs.io/en/latest/format.html). An example is shown belo
 
 ```python
 def IPB98y2(
-    I_p: float, B_t: float, P_sep: float, n19: float, R_0: float, A: float, kappa: float
+    I_p: float, B_t: float, P_sep: float, n: float, R_0: float, A: float, kappa: float
 ) -> float:
   """
   ITER IPB98(y, 2) Confinement time scaling [2]
 
   Parameters
   ----------
-  I_p
+  I_p:
       Plasma current [A]
-  B_t
+  B_t:
       Toroidal field at R_0 [T]
-  P_sep
+  P_sep:
       Separatrix power [W]
-  n
+  n:
       Line average plasma density [1/m^3]
-  R_0
+  R_0:
       Major radius [m]
-  A
+  A:
       Aspect ratio
-  kappa
+  kappa:
       Plasma elongation
 
   Returns
   -------
-  tau_E, the energy confinement time [s]
-
+  tau_E:
+      Energy confinement time [s]
```

Comment:
tau_E will be seen as the type I think? Need to check the sphinx docs
```suggestion
      Energy confinement time [s]
```

Reply:
bugger, you're right